banzai_floyds/extract.py

```diff
@@ -1,183 +1,34 @@
 from banzai.stages import Stage
 import numpy as np
-from astropy.table import Table, vstack
-from banzai_floyds.matched_filter import optimize_match_filter
-from numpy.polynomial.legendre import Legendre
-from banzai_floyds.utils.fitting_utils import gauss, fwhm_to_sigma
-from astropy.modeling import fitting, models
+from astropy.table import Table
 from banzai.logs import get_logger
-from banzai_floyds.utils.profile_utils import profile_fits_to_data
 
 
 logger = get_logger()
 
 
-def profile_gauss_fixed_width(params, x, sigma):
```

Comment:
I refactored the profile fitting and background fitting into their own files to make the code easier to find in the future.